.github/workflows/sanitizer-build-and-test.yaml

```diff
@@ -35,16 +35,16 @@ env:
   # static sanitizer clang LDFLAGS or dynamic sanitizer gcc LDFLAGS
   LDFLAGS: "-fsanitize=address,undefined"
   ASAN_OPTIONS: suppressions=${{ github.workspace }}/scripts/suppressions/suppr_asan.txt
-    detect_odr_violation=0 log_path=${{ github.workspace }}/sanitizer/sanitizer
+    detect_odr_violation=0 log_path=${{ github.workspace }}/sanitizer
```

Comment:
Shouldn't the `upload-artifact` recurse in this case and upload the subdirectory as well? Anyway, I think this change is OK.